dist/scripts/rx-data-table.js

```diff
@@ -408,16 +408,27 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
 
             scope.scoreTooltipTemplate = function (row) {
                 var template = [
-                    'Waiting on Racker for ', $filter('rxAge')(row.score.lastResponseTime, 3, true)
+                    'Waiting on Racker for ', $filter('rxAge')(row.score.lastResponseTime, 3, true),
+                    'Score: ',
+                    '- (((',
+                    row.score.maxResponseTime,
+                    ' * ',
+                    row.score.weight,
+                    ')',
+                    ' - ',
+                    row.score.elapsedTime,
+                    ') / ',
+                    row.score.threshold,
+                    ')'
                 ];
                 return template.join(' ');
             };
 
             scope.maxResponseCalculation = function (row) {
                 var template = [
                     row.account_tier || '', //jshint ignore:line
-                    row.score.type, 'for ticket is ',
-                    $filter('rxAge')(moment().subtract(row.score.maxResponseTime, 'seconds'), 2, true)
+                    row.score.type,
+                    'for ticket is ', $filter('rxAge')(moment().subtract(row.score.maxResponseTime, 'seconds'), 2, true)
```

Comment:
Any reason these two array elements need to be on the same line while the others don't?